webview_panels/src/modules/whatsNew/whatsNew.module.scss

```
@@ -0,0 +1,810 @@
+// What's New panel — ported from the approved design mockup
+// (extension-changelog.html).
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)